themes/docsy/layouts/samples/single.html

```diff
@@ -19,9 +19,11 @@ <h1>{{ .Title }}</h1>
 		<div class="github-imported-sample">
 			{{ if .Params.readme }}
 				{{ $user_repo := (urls.Parse .Params.repo).Path }}
-				{{ $resp := getJSON (print "https://api.github.com/repos" $user_repo "/readme") }}
-				<!-- RegEx replace removes any image references which could be broken -->
-				{{ $resp.content | base64Decode | replaceRE "\\!\\[(.*)\\]\\((.*)\\)" "IMAGE" | markdownify }}
+				{{ if ne .Site.BaseURL "http://localhost:1313/developer" }}
```

Comment:
This probably works for now, but might be more future proof if we do something like:
`{{ if ne hugo.Environment "development" }}`
or
`{{ if eq (getenv "HUGO_ENV") "production" }}`

Reply:
100%, was just a quick fix to get unblocked. Is that 'environment' variable exposed? I can update the PR to use that instead.

Reply:
Looks like that worked.